From 607065b410b4768372c204ccdbbca200230fff25 Mon Sep 17 00:00:00 2001
From: Nicolas Chauvet <kwizart@gmail.com>
Date: Wed, 2 Jan 2013 22:44:33 +0100
Subject: [PATCH] Fix build with unreleased FLAC 1.3.x

The issue was initialy reported here:
http://bugzilla.redhat.com/891123

FLAC upstream has always expected to include <FLAC/ suffix
for included headers as shown in examples/c/decode/file/main.c

The FLAC suffix will not be be made available in the flac.pc
for the next release:
http://lists.xiph.org/pipermail/flac-dev/2012-April/003355.html

Signed-off-by: Jean-Baptiste Kempf <jb@videolan.org>
---
 modules/codec/flac.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules/codec/flac.c b/modules/codec/flac.c
index 4cfbfa9..543e8bb 100644
--- a/modules/codec/flac.c
+++ b/modules/codec/flac.c
@@ -37,8 +37,8 @@
 #include <vlc_plugin.h>
 #include <vlc_codec.h>
 
-#include <stream_decoder.h>
-#include <stream_encoder.h>
+#include <FLAC/stream_decoder.h>
+#include <FLAC/stream_encoder.h>
 
 #include <vlc_block_helper.h>
 #include <vlc_bits.h>
-- 
1.7.6.6.GIT

